### PR TITLE
docs: restructure the administration section

### PR DESCRIPTION
**Pull request by ennru**
* A few shell commands were lacking the newline `\`
* Rewrote a few sentences for hopefully better readability.

### CODE REVIEW

**Comment by ennru** [comment]
I now split the installation into more pages to make it more of a step by step experience.

* Removed the `use.strimzi` flag
* Show the `--reuse-values` in upgrade

File: docs/docs-source/docs/modules/administration/pages/index.adoc [hunk 1]
```diff
@@ -7,9 +7,17 @@ include::ROOT:partial$include.adoc[]
 
 This chapter describes common Cloudflow and Kubernetes administration tasks.
 
-. xref:installing-cloudflow.adoc[Installing Cloudflow]
-. xref:how-to-install-and-use-strimzi.adoc[Installing Kafka with Strimzi]
-. xref:installing-spark-operator.adoc[Adding Spark Support]
-. xref:installing-flink-operator.adoc[Adding Flink Support]
-. xref:upgrading-cloudflow.adoc[Upgrading Cloudflow]
-. xref:providing-external-access-to-cloudflow-services.adoc[Providing External Access to Cloudflow Services]
+It contains the steps install Cloudflow, using the `Helm` and `kubectl` command-line tools.
```
Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/administration/pages/index.adoc:10]:
```suggestion
It contains the steps to install Cloudflow, using the `Helm` and `kubectl` command-line tools.
```

File: docs/docs-source/docs/modules/administration/pages/installing-enterprise.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+This instruction shows how to install the Cloudflow Enterprise components. These components are available to https://www.lightbend.com/lightbend-subscription[Lightbend Subscription] customers.
+
+== Installing Enterprise components
+
+Before installing, you need to make sure you have the credentials required to download the cloudflow-enterprise components from the Lightbend commercial repository.
+
+Add the Lightbend Helm repository and update the local index:
+
+  helm repo add lightbend https://repo.lightbend.com/helm-charts/
```
Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/administration/pages/installing-enterprise.adoc:17]: These are now moved to cloudflow-helm-charts

Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/administration/pages/installing-enterprise.adoc:17]:
```suggestion
  helm repo add cloudflow-helm-charts https://lightbend.github.io/cloudflow-helm-charts/
```

Comment (RayRoestenburg) [reply docs/docs-source/docs/modules/administration/pages/installing-enterprise.adoc:17]: (Strange, since this was updated in master)

Comment (RayRoestenburg) [reply docs/docs-source/docs/modules/administration/pages/installing-enterprise.adoc:17]: (or maybe I missed it)

File: docs/docs-source/docs/modules/get-started/pages/exercise-example.adoc [hunk 3]
```diff
@@ -98,11 +98,18 @@ Containers:
...
+NAME                                     TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)    AGE
+sensor-data-scala-http-ingress-service   ClusterIP   10.0.22.206   <none>        3000/TCP   1m
+----
+
+Setup local port forwarding for the Pod port
 
 [source, text]
 ----
-$ kubectl port-forward svc/sensor-data-http-ingress-service -n sensor-data-scala 3000:3000
+$ kubectl port-forward svc/sensor-data-scala-http-ingress-service -n sensor-data-scala 3000:3000
```
Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/get-started/pages/exercise-example.adoc:112]: 👍

**Review by RayRoestenburg — APPROVED** [review]

**Pull request merged**